src/components/layout/Header.tsx

```diff
@@ -26,10 +138,12 @@ export function Header() {
   const [menuOpen, setMenuOpen] = useState(false);
   const [isSignUpOpen, setIsSignUpOpen] = useState(false);
   const toggleMenu = useCallback(() => setMenuOpen((prev) => !prev), []);
-  const { isMobile } = useScreenSize();
   const { user } = useAuth();
   const { headerTitle, showTitle, setShowTitle } = useHeaderTitle();
 
+  // Radix menu for React doesn't have a way to turn this off, simulate it by a really long delay
+  const disableOpenOnHoverDelay = 999999;
```

Comment:
This is to mimic current behavior of not open submenu on hover, we could discuss whether we want that or not.

Reply:
Indeed and I don't feel strongly about it in either direction which is why I didn't change it. I figure we could leave it as it is for now and change it later as a separate change.